front/pages/w/[wId]/builder/assistants/new.tsx

```diff
@@ -39,6 +44,9 @@ export const getServerSideProps: GetServerSideProps<{
   const allDataSources = await getDataSources(auth);
   const allDustApps = await getApps(auth);
 
+  assert(BUILDER_FLOWS.includes(context.query.flow as BuilderFlow));
```

Comment:
We don't generally assert in getServerSideProps.

Let's validate and set or default to something. You can use this trick to have both an array and a type: https://github.com/dust-tt/dust/blob/main/front/pages/w/%5BwId%5D/assistant/gallery.tsx#L25-L30

Reply:
yes this is how BuilderFlow is defined :)